www/docs/resources.md

```diff
@@ -81,7 +81,7 @@ POST /v1/streams
 DELETE /v1/streams/:uuid
 ```
 
-The `POST` route instantiates a resource according to the JSON-encoded request body (consisting of the resource identifier and any parameters, structured as `{ resource: string; params: { [param: string]: string } }`) and returns a UUID identifying the resource, which can then be used in a query to the streaming API.
+The `POST` route instantiates a resource according to the JSON-encoded request body (consisting of the resource identifier and any parameters, structured as `{ resource: string; params: { [param: string]: any } }`) and returns a UUID identifying the resource, which can then be used in a query to the streaming API.
```

Comment:
My thinking was that this is talking about string-encoded JSON payloads on HTTP requests, not JavaScript values/types -- so "any" felt more general for e.g. web service clients not using JavaScript.

Reply:
The implementation will treat the body assuming the type with `Json` instead of `any`, and in particular pass the params to `skjson.exportJSON`, so the body will have to be JSON independently of the language users write in. But maybe just saying that the body is JSON-encoded is enough to imply that the values of params must also be JSON. I don't know, but what is written here is a typescript type, so it seems that it would be clearest to give the correct one here (and let users of other languages infer the gap themselves) or else document the required form of the body using just JSON (schemas?) and no typescript types.

Reply:
We could (and probably should) just define some [proper schema](json-schema.org)

In this case, though, I don't think there's any meaningful difference between `Json` and `any` -- both allow for any text-encodable JSON value. So the question is which one serves better as documentation, which for me would be `any` (expecting more people to be familiar with it than our `Json` as a \top type)

Reply:
Ok. To me `any` includes things like closures and objects with methods, so I find it confusing here, but maybe just me.